optuna/study.py

```diff
@@ -18,6 +18,7 @@
 from optuna import samplers
 from optuna import storages
 from optuna import trial as trial_module
+from optuna._callbacks import MaxTrialsCallback
```

Comment:
To fix warning for unused declaration.
```suggestion
from optuna._callbacks import MaxTrialsCallback  # NOQA
```

Reply:
Thanks! That was what that was for... 😅